README.md

```diff
@@ -3,54 +3,100 @@
 The `py-allotax` implements a python interface to the `allotaxonometer` library. The tool here provides a way for users to input data and arguments and receive back a saved plot! The tool is designed to be used in a command line or in a python notebook in a few lines of code (see usage instructions at the bottom).
 
 Table of contents:
-- [Installation steps](#installation-steps)
+- [Package Build](#package-build)
 - [Usage instructions](#usage-instructions)
+- [Developer Notes](#developer-notes)
 - [Frequent questions or issues](#frequent-questions-or-issues)
 - [Repo structure notes](#repo-structure-notes)
 - [Resources](#resources)
 
 
+
 ## Installation steps
 
+1. Install `python3.11` or greater.
+
 1. If JavaScript tool installs are needed (never used or installed `npm`, `nvm`, `node`):
     1. Here are the recommended [steps to install `nvm`](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating). `nvm` is a node version manager that streamlines installing the other 2.
     - Otherwise (not recommended): [steps to individually install `node` and `npm`](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
 1. Once you have `nvm`, install the latest of both `node` and `npm` with:
-    ```shell
+    ```bash
     nvm install --lts
     ```
-
+1. Ensure docker or podman is installed on your machine.
```

Comment:
Fun story, I had to help a very smart person with installing docker on their local machine running on DOS. I ended up running code for her on my machine instead.